### PR TITLE
Shipping

### PR DESCRIPTION
**Pull request by sorokinvj**
# Motivation
We need to charge users for shipping:

| Quantity | Europe | World | US | Portugal |
| --------- | ------- | ----- | --- | -------- |  
|1-3 | 5,35 euro | 6,30 |  6,55 | 3,35 |
| 3-10 |  6,20 euro | 7,30 | 8,75 | 3,95 |
| 10+ | 8 euro | 10, 85 | 12,85 | 6,5 |

# Solution
- Shipping info  is collected by Stripe
- Hygraph model `Order` updated with new fields - shippingInfo and shippingCost
- on the Success page we show shipping info and ask user to contact us if it is incorrect
- FREE shipping on all orders total > 50 euro

### CODE REVIEW

File: components/CartItem/CartItem.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ interface Props {
 
 export const CartItem: React.FC<Props> = ({ item }) => {
   const { submissionLoading } = useSubmissionState()
-  const { removeItem, updateItemQuantity } = useCart()
```
Comment (sorokinvj) [inline components/CartItem/CartItem.tsx:21]: leftover from previous pr

File: lib/create-order.ts [hunk 2]
```diff
@@ -1,40 +1,63 @@
...
+                  id: item.price.product.metadata.productId
+                }
+              }
+            }))
           },
-        })),
-      },
-    },
-  });
+          shippingInfo: {
```
Comment (sorokinvj) [inline lib/create-order.ts:46]: this is a very popular and important pattern that uses [object spreads](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax). Take a look at the docs if you feel uncomfortable with it

File: styles/index.css [hunk 3]
```diff
@@ -3,3 +3,12 @@
 @tailwind components;
 
 @tailwind utilities;
+
+@layer components {
+  a.colored {
```
Comment (sorokinvj) [inline styles/index.css:8]: this is how you implement tailwind custom classes

**Review by valieff — APPROVED** [review]
great work!

**Pull request merged**